skiko/src/commonTest/kotlin/org/jetbrains/skia/ParagraphTest.kt

```diff
@@ -49,23 +46,49 @@ class ParagraphTest {
     }
 
     @Test
-    @SkipJsTarget
-    @SkipWasmTarget
     @SkipNativeTarget
```

Comment:
Not sure what changed, but `@SkipNativeTarget` is not needed anymore too - the test passed on macos and ios simulator.

Reply:
Removed that annotation